### PR TITLE
Run the SEIR model from Python using OpenCOR as a black box

### PR DESCRIPTION
**Pull request by agarny**
This is the first step. The next one is to use the MoH data using @mahyar-osn's Python script.

Otherwise, note that the `seir` command works fine on macOS, but for technical reasons it does not on Windows and Linux...! On Windows, I am getting:

```
C:\Users\Alan\Desktop\OpenCOR-2020-04-07-Windows>OpenCOR -c PythonShell Python\Scripts\seir.py 
Traceback (most recent call last):
  File "Python\Scripts\seir.py", line 73, in <module>
    main()
  File "Python\Scripts\seir.py", line 38, in main
    import matplotlib.pyplot as plt
  File "C:\Users\Alan\Desktop\OpenCOR-2020-04-07-Windows\Python\lib\site-packages\matplotlib\pyplot.py", line 2359, in <module>
    switch_backend(rcParams["backend"])
  File "C:\Users\Alan\Desktop\OpenCOR-2020-04-07-Windows\Python\lib\site-packages\matplotlib\pyplot.py", line 225, in switch_backend
    backend_mod = importlib.import_module(backend_name)
  File "C:\Users\Alan\Desktop\OpenCOR-2020-04-07-Windows\Python\Lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "C:\Users\Alan\Desktop\OpenCOR-2020-04-07-Windows\Python\lib\site-packages\matplotlib\backends\backend_tkagg.py", line 1, in <module>
    from . import _backend_tk
  File "C:\Users\Alan\Desktop\OpenCOR-2020-04-07-Windows\Python\lib\site-packages\matplotlib\backends\_backend_tk.py", line 6, in <module>
    import tkinter as tk
  File "C:\Users\Alan\Desktop\OpenCOR-2020-04-07-Windows\Python\Lib\tkinter\__init__.py", line 36, in <module>
    import _tkinter # If this fails your Python may not be configured for Tk
ModuleNotFoundError: No module named '_tkinter'
```

while on Linux, I am getting:

```
QWidget: Cannot create a QWidget without QApplication
Aborted (core dumped)
```

We really need to have a "proper" OpenCOR Python package!

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+plt.plot(voi.values(), R_t.values(), label=R_t.name())
+plt.plot(voi.values(), R_u.values(), label=R_u.name())
+plt.legend(loc='center left')
+
+plt.subplot(4, 1, 4)
+plt.plot(voi.values(), IFR.values(), label=IFR.name())
+plt.legend(loc='center left')
+plt.xlabel('time (day)')
+
+plt.show()
```
Comment (mahyar-osn) [inline src/main.py]: Add trailing line.

**Review by mahyar-osn — APPROVED** [review]
Looks good overall. 

With regards to _tkinter problem in Windows, we might be able to have a quick and dirty workaround by adding `matplotlib.use('agg')` after `import matplotlib.pyplot as plt`.

File: src/seir.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
+import opencor as oc
+
+
+def main():
+    # Open and run a 300-day simulation using the SEIR model.
+
+    simulation = oc.open_simulation(
+        'https://models.physiomeproject.org/workspace/5d4/rawfile/b973009caf430b1636cfd1133a23bfd570a85691/COVIDSim.cellml')
```
Comment (nickerso) [inline src/seir.py:8]: should be loading a SED-ML with the appropriate simulation algorithm defined, etc.

Comment (agarny) [reply src/seir.py:8]: Doh, good point! Going to do that.

Comment (agarny) [reply src/seir.py:8]: Done in PR #2.

**Pull request merged**